charts/prometheus-adapter/templates/psp.yaml

```
@@ -4,9 +4,14 @@ apiVersion: policy/v1beta1
 kind: PodSecurityPolicy
 metadata:
   name: {{ template "k8s-prometheus-adapter.fullname" . }}
-  {{- if .Values.customAnnotations }}
+  {{- if or .Values.customAnnotations .Values.pspAnnotations }}
```

Probably better to merge them together to reduce lines and in case an annotation is repeated in both.

```suggestion
  {{- with (merge .Values.customAnnotations .Values.pspAnnotations) }}
```